db/versions/0017.yml

```diff
@@ -0,0 +1,242 @@
+version: 17
+migrationScript: 0017-migration.sql
+downgradeScript: 0017-downgrade.sql
+methods:
+  wmworker_pool_errors_entities_load:
+    description: See taskcluster-lib-entities
+    mode: read
+    serviceName: worker_manager
+    args: partition_key text, row_key text
+    returns: table (partition_key_out text, row_key_out text, value jsonb, version integer, etag uuid)
+    body: |-
+      begin
+        return query
+        select
+          wmworker_pool_errors_entities_load.partition_key,
+          wmworker_pool_errors_entities_load.row_key,
+          entity_buf_encode(
+            jsonb_build_object(
+              'PartitionKey', encode_string_key(error_id),
```

Comment:
This should be `worker_pool_id` rather than `error_id`. This comment applies to the other places in this file where the partition key is specified.